### ISSUE TITLE
Citizenship message read three times

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5423]
## ❌ Current behaviour (issue)
The form label information message for citizenship status was repeated three times. It would be nice if it could just be repeated one time for the field component.

### AT Configuration
- OS: Windows
- Assistive technology: JAWS
- Browser: Edge
### How to reproduce
1.	Open JAWS
2.	Navigate using your keyboard to the Profile and then the **About Me** Section 
3.	Tab down to the **Citizenship** component. 
4.	Enter information and note how many times the Citizenship, PR requirement is read by the screen reader.
### Fable Video Clip
[Citizenship Bug](https://app.makeitfable.com/share/63cae417a597030013974283)
## ✔ Expected behaviour (fix)
Citizenship, PR requirement notification is only read once for the screen reader.
### Acceptance Criteria
- [ ] Citizenship, PR requirement notification is only read once for the screen reader.
## Additional information (optional)
1. How does it impact users? Removes a usability irritant for AT users.

**Comment by mnigh** [comment]
I tried reproducing this on one of the listed configurations in this issue's description but could not. My configuration was:

- OS: macOS 12.6.3 (21G419)
- Browser: Safari Version 16.3 (17614.4.6.11.4, 17614)
- Assistive technology: VoiceOver

Based on the screen recording information in Fable, the technology used by the tester is, so perhaps this issue's description needs to be updated to reflect this:

- OS: Windows
- Assistive technology: JAWS
- Browser: Edge

@esizer are you able to test on Windows in Edge with JAWS?

**Comment by RM-1978** [comment]
Thanks @Matt! This issue was identified by someone with a MacOS system in the compatibility test and another tester in the recorded self-test. That said, I am glad that it looks like the issue that the MacOS user identified on UAT in early Dec cannot be reproduced.  I am happy to update the  configuration as you receommended!

**Comment by esizer** [comment]
> @esizer are you able to test on Windows in Edge with JAWS?

I am unable to test with JAWS specifically since it requires a license 👎

**Comment by petertgiles** [comment]
Eric waiting for JAWS license
